superset-frontend/src/dashboard/components/gridComponents/ChartHolder.jsx

```diff
@@ -159,9 +171,14 @@ class ChartHolder extends React.Component {
     });
   }
 
+  handleFullSize() {
+    const flag = !this.state.isFullSize;
+    this.setState(() => ({ isFullSize: flag, resizeEvent: true }));
+  }
+
   render() {
     const { isFocused } = this.state;
-
+    const PADDING = 32;
```

Comment:
prefer defining constants at the top of the file instead of in the render function. Also, could use a more descriptive variable name

Reply:
Changed, I reuse another constant at the top of file c0526bd607a84ebc9e32f7d4676af14a06a2624b